Gemfile

```diff
@@ -56,6 +56,8 @@ gem 'octokit', '~> 4.8.0', :require => false
 gem 'faraday', '~> 0.9.1'
 gem 'faraday-http-cache', '~> 2.0.0'
 
+gem 'xmlrpc'
```

Comment:
Don't do this in this PR.  I have made a larger comment here on why:

https://github.com/ManageIQ/miq_bot/issues/427#issuecomment-443352344

Reply:
👍 Thanks @NickLaMuro

Reply:
@juliancheal Okay, looked into this further, since @bdunne questioned my sanity in `gitter`™ a few weeks ago:

https://gitter.im/ManageIQ/miq_bot?at=5bf59210a115c91ef76eb9e4

and that got me thinking I might be wrong.  Turns out, while on `master` of `active_bugzilla`, the PR from LJ to improve performance for the `XMLRPC::Client` was merged by @Fryguy :

https://github.com/ManageIQ/active_bugzilla/pull/49

The most recent version on rubygems of `active_bugzilla` is actually from 1.5 years prior...

https://rubygems.org/gems/active_bugzilla

So yeah, you are right in saying that this will get things working...

* * *

That said, that probably shouldn't stop us from moving forward and cleaning off a bit of dust and such from `active_bugzilla`, but that probably isn't necessary for #427 to be resolved.

[#blameFryguy](http://fingerofblame.org/blame/QEZyeWd1eQ==)